src/psyclone/gocean1p0.py

```diff
@@ -892,8 +896,25 @@ def reference_accesses(self, var_accesses):
         :type var_accesses: \
             :py:class:`psyclone.core.access_info.VariablesAccessInfo`
         '''
+
+        # Grid properties are accessed using one of the fields. This stores
+        # the field used to avoid repeatedly determining the best field:
+        grid_for_property = None
         for arg in self.arguments.args:
-            var_accesses.add_access(arg.name, arg.access, self)
+            if arg.type == "grid_property":
+                if grid_for_property is None:
```

Comment:
This should be `if not grid_for_property:`

Reply:
I thought I had pylint telling me in the past that I needed 'is' for None comparisons. Afaik there is a slight difference (since 'if not' would also trigger when the value is 0). Doesn't matter here, and indeed pylint does not complain about if not ... so done.